index.html

```diff
@@ -57,131 +60,187 @@ <h3>I'm a Software Developer</h3>
-          the industry's standard.</p>
-      </div>
-      <div class="links">
-        <ul>
-          <li>HTML</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-      </div>
-      <div class="buttons">
-        <a href="#">See project</a>
-      </div>
-    </section>
-    <section class="img_p_holder">
-      <div class="mp_stories">
-        <h2>Professional Art<br> Printing Data</h2>
-        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
-          the industry's standard.</p>
-      </div>
-      <div class="links">
-        <ul>
-          <li>HTML</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-      </div>
-      <div class="buttons">
-        <a href="#">See project</a>
-      </div>
-    </section>
-    <section class="img_p_holder">
-      <div class="mp_stories">
-        <h2>Professional Art<br> Printing Data</h2>
-        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
-          the industry's standard.</p>
-      </div>
-      <div class="links">
-        <ul>
-          <li>HTML</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-      </div>
-      <div class="buttons">
-        <a href="#">See project</a>
-      </div>
-    </section>
-    <section class="img_p_holder">
-      <div class="mp_stories">
-        <h2>Professional Art<br> Printing Data</h2>
-        <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required. has been
-          the industry's standard.</p>
-      </div>
-      <div class="links">
-        <ul>
-          <li>HTML</li>
-          <li>Bootstrap</li>
-          <li>Ruby</li>
-        </ul>
-      </div>
-      <div class="buttons">
-        <a href="#">See project</a>
+    <section class="recent_works">
+      <div class="wrapper">
+        <div class="title">
+          <h1>My Recent Works</h1>
+          <hr class="line">
+        </div>
+        <div class="recent_summary">
+          <div class="place_holder">
+            <img src="/static/images/ph1.png" alt="place_holder">
+          </div>
+          <div class="stories">
+            <div class="mp_stories">
+              <h2>Multi-Post Stories</h2>
+              <p>A daily selection of selection of privately personalized reads; no accounts or sign-ups required: has
+                been
+                the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+                text.
+              </p>
+            </div>
+            <div class="links">
+              <ul>
+                <li>CSS</li>
+                <li>HTML</li>
+                <li>Bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+            </div>
+            <div class="buttons btn_width_wrap">
+              <a href="#">See project</a>
+            </div>
```

Comment:
I have a couple of suggestions to enhance the layout of your portfolio:

- [ ] It would be wonderful if you could align the `See Project` button of your first card to the bottom of the card. 🎯
- [ ] Additionally, it would be great if **you could apply the correct hover effect to all the remaining cards.** When you **hover over a card, the description and dark opacity should fade away, and a "See Project" button should appear at the bottom of the image,** just like in the first card. 🔍